.retest/recheck.ignore.js

```diff
@@ -0,0 +1,46 @@
+var allowedPxDiff = 5;
```

Comment:
I'm not that familiar with JS, but aren't these [constants](https://stackoverflow.com/a/130399)? So if we don't want to rely on ES2015, shouldn't this be something like `ALLOWED_PIXEL_DIFF`? And the other variables/constants?

Reply:
Renamed...